zarr/store/zipfile.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, print_function, division
+
+
+from zarr.store.base import ArrayStore
+
+
+class ZipFileStore(ArrayStore):
+
+    def __init__(self, path):
+        self._path = path
+        # TODO
+
+    @property
+    def meta(self):
+        pass
+
+    @property
+    def data(self):
+        pass
+
+    @property
+    def attrs(self):
+        pass
+
+    @property
+    def cbytes(self):
+        pass
+
+    @property
+    def initialized(self):
+        pass
```

Comment:
Yes, you could implement a generic array store class that allows the user to pass in a Mapping for the metadata and a MutableMapping for the compressed chunk data (and optionally a MutableMapping for user attributes) allowing in principle any combination of storage for these three elements.